students/gyuchan/users/views.py

```diff
@@ -33,5 +32,26 @@ def post(self, request):
                 phone_number = user_phone_number
             )
             return JsonResponse({'message':'CREATED'}, status=201)
+        except KeyError:
+            return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+
+class LogInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            
+            user_id       = data['id']
```

Comment:
```suggestion
            user_email       = data['email']
```
지금 받아와야 하는 값은 user의 id가 아니라 email 입니다.